test/shopify-cli/commands/connect_test.rb

```diff
@@ -5,7 +5,80 @@ module Commands
     class ConnectTest < MiniTest::Test
       include TestHelpers::Partners
 
-      def test_run
+      def test_connect_asks_project_type_and_writes_yml_when_no_project_exists
+        ShopifyCli::Project.stubs(:has_current?).returns(false)
+        Resources::EnvFile.expects(:parse_external_env).returns({})
```

Comment:
Just wondering if there's a reason to use `expects` here but `stubs` on the line above?

Reply:
`Project.has_current?` gets called so many times that it was easier to stub it rather than set it to `.times(5)` (or whatever the number is). As for the `EnvFile`, I like having more `expects` when I can so that we know the code is executing as expected since we're not doing any `asserts`.